<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Pranab" ignored="0" name="Application_Test" tests="3" failures="1" timestamp="13 Apr 2017 07:07:57 GMT" time="0.064" errors="0">
  <testcase name="doLogin" time="0.015" classname="TestCases.Application_Test">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error message ">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Error message 
	at TestCases.Application_Test.doLogin(Application_Test.java:46)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="doPasswordchange" time="0.0" classname="TestCases.Application_Test">
    <skipped/>
  </testcase> <!-- doPasswordchange -->
  <testcase name="doLogout" time="0.0" classname="TestCases.Application_Test">
    <skipped/>
  </testcase> <!-- doLogout -->
</testsuite> <!-- Application_Test -->
